packages/wallet/src/containers/initialized.tsx

```diff
@@ -12,12 +12,12 @@ interface Props {
 class WalletInitializedContainer extends PureComponent<Props> {
   render() {
     const { state } = this.props;
-    if (!state.currentProcessId) {
-      return <LandingPage />;
-    } else {
+
```

Comment:
I think prettier needs to be run on this file. You can do `yarn prettier:write` or just use the `prettier` vs code extension.